ui/mirage/scenarios/default.js

```diff
@@ -2,6 +2,7 @@ import config from 'nomad-ui/config/environment';
 
 const withNamespaces = getConfigValue('mirageWithNamespaces', false);
 const withTokens = getConfigValue('mirageWithTokens', true);
+const withRegions = getConfigValue('mirageWithRegions', false);
```

Comment:
Might want to add this setting to config/environment.js alongside the other Mirage options?

Reply:
Heh, definitely. I added it locally, but I so rarely check in changes to the environment file that I forgot I had changes worth committing. Good spot.